lib/tessel/deploy.js

```diff
@@ -334,13 +334,14 @@ actions.tarBundle = function(opts) {
 
           return rules.concat(patterns);
         }, []).map(function(rule) {
-
+          // Ensure it uses proper paths for OS
+          rule = path.normalize(rule);
           if (rule[rule.length - 1] === '/') {
-            rule += '**/*.*';
+            rule += path.normalize('**/*.*');
           }
 
           if (rule[rule.length - 1] !== '*' && rule.indexOf('.') === -1) {
-            rule += '/**/*.*';
+            rule += path.normalize('/**/*.*');
           }
 
```

Comment:
None of these are necessary, `glob` will do the normalization for you. Please revert (it's also an unnecessary conflict with #435 )